README.md

```diff
@@ -195,6 +200,51 @@ This tells us that PostgreSQL is "_listening_" on TCP Port `5432`
 
 With all those "pre-flight checks" performed, let's get _going_!
 
+## First _Run_ the _Finished_ App
+
+_Before_ you attempt to build the Chat App from scratch,
+clone and run the _finished_ working version
+to get an idea of what to expect.
+
+### Clone the Project:
+
+In your terminal run the following command to clone the repo:
+
+```sh
+git clone git@github.com:dwyl/phoenix-chat-example.git
+```
+
+### Install the Dependencies
+
+Change into the `phoenix-chat-example` directory
+and install both the `Elixir` and `Node.js` dependencies
+with this command:
+
+```sh
+cd phoenix-chat-example
+mix setup
+```
+
+<!-- ### TODO: Add auth step! -->
+
+
+
+### Run the App
+
+Run the Phoenix app with the command:
+
+```sh
+mix phx.server
+```
+
+# TODO: Add GIF of Finished App
```

Comment:
Do we want to keep this TODO as a reminder or should we comment it for now?

Reply:
@SimonLab Yes, we need a GIF of the Chat App running on Localhost ... 
I just ran the app on my mac and noticed that the CSS is "off", so I'm fixing it now.

Reply:
![phoenix-chat-localhost-demo](https://user-images.githubusercontent.com/194400/84200400-a2bbe900-aa9e-11ea-95fd-4e9b2dde4820.gif)

Reply:
That previous GIF was _tiny!_ (_click on it and you will see_)
This one is full page width:
![phoenix-chat-localhost-demo-optimised](https://user-images.githubusercontent.com/194400/84203142-d861d180-aaa0-11ea-8f10-abff03c3b4d2.gif)

Reply:
GIF added to `README.md` ✅